lib/screens/play/card/card_pile.dart

```diff
@@ -1,33 +1,48 @@
 import 'package:flutter/material.dart';
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
 
 class CardPile extends StatelessWidget {
```

Comment:
Are we likely to have any other card piles than 'deck' and 'discard'?

Reply:
Pass!